tests/test_baseconf.py

```diff
@@ -34,12 +34,13 @@ def test_baseconf_init(baseconf_gen):
 
 
 def test_is_embargoed(baseconf_gen):
-    assert baseconf_gen.filter_embargoed()
+    assert baseconf_gen.filter_embargoed("None")
 
     baseconf_gen.settings["PUBLIC_CLOUD_SOMETHING"] = ""
 
-    assert baseconf_gen.filter_embargoed()
+    assert baseconf_gen.filter_embargoed("None")
 
     baseconf_gen.settings = {}
 
-    assert baseconf_gen.filter_embargoed() is False
+    assert baseconf_gen.filter_embargoed("Noone") is False
+    assert baseconf_gen.filter_embargoed("Azure-test")
```

Comment:
I guess this line is the only real test addition for the new change of behaviour, right?

Reply:
yes, real change is only `or any(
            flavor.startswith(s) for s in ("Azure", "EC2", "GCE")` , rest is only call semantic and remove one layer of indentation in aggregates :D